packages/libp2p/src/connection-manager/constants.browser.ts

```diff
@@ -16,6 +11,11 @@ export const MIN_CONNECTIONS = 5
 export const MAX_CONNECTIONS = 100
 
 /**
- * @see https://libp2p.github.io/js-libp2p/interfaces/index._internal_.ConnectionManagerConfig.html#autoDialConcurrency
+ * @see https://libp2p.github.io/js-libp2p/interfaces/index._internal_.ConnectionManagerConfig.html#maxParallelDials
+ */
+export const MAX_PARALLEL_DIALS = 50
+
+/**
+ * @see https://libp2p.github.io/js-libp2p/interfaces/libp2p.index.unknown.ConnectionManagerInit.html#autoDialPeerRetryThreshold
  */
-export const AUTO_DIAL_CONCURRENCY = 10
+export const AUTO_DIAL_PEER_RETRY_THRESHOLD = 1000 * 60 * 7
```

Comment:
Why is this being increased to 7 minutes?

Reply:
To be slightly longer than the WebTransport error timeout - see https://bugs.chromium.org/p/chromium/issues/detail?id=1473980

Granted this could be a separate PR.